src/containers/shared/transactionUtils.ts

```diff
@@ -79,17 +79,6 @@ export const CURRENCY_OPTIONS = {
   maximumFractionDigits: 8,
 };
 
-export const DATE_OPTIONS = {
-  hour: 'numeric',
-  minute: 'numeric',
-  second: 'numeric',
-  year: 'numeric',
-  month: 'long',
```

Comment:
This is different than the others - most of the other ones say `numeric` for the month